WordPress/Classes/ViewRelated/Blaze Campaigns/BlazeCampaignsViewController.swift

```diff
@@ -26,13 +34,39 @@ final class BlazeCampaignsViewController: UIViewController, NoResultsViewHost, B
         return tableView
     }()
 
+    private lazy var promoteButton: UIButton = {
+        let button = UIButton(type: .system)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.addTarget(self, action: #selector(buttonCreateCampaignTapped), for: .touchUpInside)
+
+        var configuration = UIButton.Configuration.filled()
```

Comment:
👀

Reply:
I needed to set this to .filled instead of .plain in order to configure the baseBackgroundColor

Reply:
No, just great to see someone else using these new APIs 😅
Using `.filled` seems to be a good way to go.